commands-extender/commands-provider.ts

```diff
@@ -31,31 +60,24 @@ class DynamicItemIndexCommandProvider implements CommandProvider {
             commands.push(previewCommand);
         }
 
+        // return an observable here, because this might be a time consuming operation
```

Comment:
*return an observable here, because this might be a time consuming operation*

| it is correct but still a bit unclear 

*The method shall return Observable as the actions menu generation might be a time consuming operation*